src/appengine/handlers/home.py

```diff
@@ -60,9 +68,22 @@ def get_single_fuzz_target_or_none(project, engine_name):
   return fuzz_target_name
 
 
+def get_introspector_index():
+  """Return introspector projects status"""
+  introspector_bucket = storage.Client().get_bucket(INTROSPECTOR_BUCKET)
```

Comment:
Missed this: we should be using google_cloud_utils.storage.read_data for this instead. 

(grep for examples for how to call it)

Reply:
Done.